web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -40,19 +41,24 @@ import { noop } from "~/utils";
  */
 
 /**
- * Form for selecting the installation device
+ * Form for selecting the installation device.
```

Comment:
NP: no strong opinion about it, but we can save one character by omitting the dot at the end of sentence in documentation. Especially when documenting paramos.

Reply:
Now you changed my mind too :P